tests/unittests/executor/test_futures.py

```diff
@@ -104,7 +104,7 @@ def wait_timeout(self, executor):
 
         future.wait(0.01)
 
-        # SingleExecutor is not truely async so 
+        # SingleExecutor is not truely async so
         # results are always ready after a wait
         if executor != SingleExecutor:
```

Comment:
`not isinstance(executor, SingleExecutor)`?

Reply:
Ah, it was done already. Sorry! I was behind in the commits history.

Reply:
already fixed